src/dbnode/runtime/runtime_options.go

```diff
@@ -38,6 +38,10 @@ const (
 	// DefaultBootstrapConsistencyLevel is the default bootstrap consistency level
 	DefaultBootstrapConsistencyLevel = topology.ReadConsistencyLevelMajority
 
+	// DefaultIndexDefaultQueryTimeout is the hard timeout value to use if none is
+	// specified for a specific query, zero specifies no timeout.
+	DefaultIndexDefaultQueryTimeout = time.Minute
```

Comment:
What value are you seeing in prod? I would have thought 15->30 would be a more reasonable default

Reply:
Yeah, it could be, I just don't want to necessarily break some users existing queries.  I could be convinced using 30s instead though for sure.

Reply:
Yup, I opened here:
https://github.com/m3db/m3/issues/1220